packages/forms-web-app/src/dynamic-forms/middleware/utils.js

```diff
@@ -30,4 +31,24 @@ function getAddMoreIfPresent(req, questionObj) {
 	return questionObj;
 }
 
-module.exports = { getAddMoreIfPresent };
+/**
+ * Converts booleans in the LPAQuestionnaireSubmission db model into yes/no strings
+ * @param {LPAQuestionnaireSubmission} dbResponse
+ */
+
+function mapDBResponseToJourneyResponseFormat(dbResponse) {
+	let convertedResponse = {};
+	for (const key of Object.keys(dbResponse)) {
+		if (dbResponse[key] === true) {
+			convertedResponse[key] = 'yes';
+		} else if (dbResponse[key] === false) {
+			convertedResponse[key] = 'no';
+		} else {
+			convertedResponse[key] = dbResponse[key];
+		}
+	}
+
+	return convertedResponse;
+}
```

Comment:
```suggestion
function mapDBResponseToJourneyResponseFormat(dbResponse) {
	return Object.entries(dbResponse).reduce(
		(acc, [key, value]) => ({
			...acc,
			[key]: (() => {
				switch (value) {
					case true:
						return 'yes';
					case false:
						return 'no';
					default:
						return value;
				}
			})()
		}),
		{}
	);
}

```

feel free to not do this, I just can't help myself

Reply:
No, you're right, made the change